react/react1/week3/todo-list/src/AddTask.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+const AddTask = ({ newTask, setNewTask, date, setDate, handleAddTask }) => {
```

Comment:
Since you are passing useState variables and they hold a value yet, you don't need to pass `newTask `and `data`. You can use just `props` instead of mentioning all the props and while referring to a specific prop, you can use `props.setNewTask`, etc.